bitnami/haproxy/README.md

```diff
@@ -0,0 +1,408 @@
+| `existingConfigmap`                  | configmap with HAProxy configuration                                                      | `""`                    |
+| `affinity`                           | Affinity for haproxy pods assignment                                                      | `{}`                    |
+| `nodeSelector`                       | Node labels for haproxy pods assignment                                                   | `{}`                    |
+| `tolerations`                        | Tolerations for haproxy pods assignment                                                   | `[]`                    |
+| `updateStrategy.type`                | haproxy statefulset strategy type                                                         | `RollingUpdate`         |
+| `priorityClassName`                  | haproxy pods' priorityClassName                                                           | `""`                    |
+| `lifecycleHooks`                     | for the haproxy container(s) to automate configuration before or after startup            | `{}`                    |
+| `extraEnvVars`                       | Array with extra environment variables to add to haproxy nodes                            | `[]`                    |
+| `extraEnvVarsCM`                     | Name of existing ConfigMap containing extra env vars for haproxy nodes                    | `nil`                   |
+| `extraEnvVarsSecret`                 | Name of existing Secret containing extra env vars for haproxy nodes                       | `nil`                   |
+| `extraVolumes`                       | Optionally specify extra list of additional volumes for the haproxy pod(s)                | `[]`                    |
+| `extraVolumeMounts`                  | Optionally specify extra list of additional volumeMounts for the haproxy container(s)     | `[]`                    |
+| `sidecars`                           | Add additional sidecar containers to the haproxy pod(s)                                   | `[]`                    |
+| `initContainers`                     | Add additional init containers to the haproxy pod(s)                                      | `{}`                    |
+
+### Other Parameters
+
+| Name                    | Description                                          | Value  |
+|-------------------------|------------------------------------------------------|--------|
+| `serviceAccount.create` | Specifies whether a ServiceAccount should be created | `true` |
+| `serviceAccount.name`   | The name of the ServiceAccount to use.               | `""`   |
+
+  log stdout format raw local0
+  maxconn 1024
+defaults
+  log global
+  timeout client 60s
+  timeout connect 60s
+  timeout server 60s
+frontend fe_main
+  bind :8080
+  default_backend be_main
+backend be_main
+  server web1 10.0.0.1:8080 check
```

Comment:
This looks wrong, could you check what happened?

Reply:
Relanunched readmenator